src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: ToDo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, id, userId } = todo;
+
+  const foundUser = usersFromServer.find(
+    checkedUser => checkedUser.id === userId,
+  );
```

Comment:
The `find` method is used correctly to locate the user, but consider adding a check to handle the case where `foundUser` might be `undefined`. This can prevent potential runtime errors if the user is not found.